tools/es5to6.js

```diff
@@ -17,91 +16,70 @@ process.on('uncaughtException', handleError);
-                )
-            );
-            process.exit(1);
-        }
-    })
-    .then(() => {
-        const moduleId = userModules.shift();
-        const unique = `${Date.now()}`.slice(-4);
-        const es5Module = path.join(
-            'static',
-            'src',
-            'javascripts-legacy',
-            moduleId
-        );
-        const es6Module = path.join('static', 'src', 'javascripts', moduleId);
-        const es6Name = moduleId.split(path.sep).join('_').replace('.js', '');
-        const branchName = `es6-${es6Name}`;
+const moduleId = userModules.shift();
+const unique = `${Date.now()}`.slice(-4);
+const es5Module = path.join('static', 'src', 'javascripts-legacy', moduleId);
 
-        const steps = {
-            'Create a branch for the conversion': `git checkout -b "${branchName}" || git checkout -b "${branchName}-${unique}"`,
-            'Move the legacy module to the new location': `mkdir -p ${path.dirname(es6Module)}; mv ${es5Module} $_; node ./tools/es5to6-remove-module.js ${moduleId}`,
-            'Commit the move': `git add .; git commit -m "move ${moduleId} from legacy to standard JS"`,
-            'Convert module to ES6': `npm run -s amdtoes6 -- -d ${path.dirname(es6Module)} -o ${path.dirname(es6Module)} -g **/${path.basename(es6Module)} `,
-            'Commit the module tranform': `git add .; git commit -m "transform ${moduleId} to ES6 module"`,
-            'Convert contents to ES6': `npm run -s lebab -- ${es6Module}`,
-            'Commit the content tranform': `git add .; git commit -m "transform ${moduleId} content to ES6"`,
-            'Fix lint errors': `npm run -s eslint-fix -- ${es6Module}`,
-            'Commit any lint fixes': `git add .; git commit -m "fix lint errors with ${moduleId} after transform to ES6"`,
-        };
+const es6Module = path.join('static', 'src', 'javascripts', moduleId);
+const es6Name = moduleId.split(path.sep).join('_').replace('.js', '');
+const branchName = `es6-${es6Name}`;
 
-        Object.keys(steps)
-            .reduce(
-                (allSteps, step, i) => allSteps.then(() => {
-                    console.log('');
-                    console.log(chalk.blue(`${i + 1}. ${step}`));
-                    console.log(chalk.dim(steps[step]));
-                    return execa
-                        .shell(steps[step].trim(), {
-                            stdio: 'inherit',
-                        })
-                        .then(() => {
-                            console.log(chalk.green('done'));
-                        })
-                        .catch(e => {
-                            console.log(chalk.red(e.stack));
-                            megalog.error(
-                                `\`${step}\` did not complete.
+const steps = {
+    'Create a branch for the conversion': `git checkout -b "${branchName}" master || git checkout -b "${branchName}-${unique}" master`,
```

Comment:
Yes, that would happen. Do you think that is a problem? Doing it the right way (checking for branch and print error message in case it's a duplicate) would be a bit of work, which I want to avoid ...